src/index.js

```diff
@@ -0,0 +1,296 @@
+      />
+      <!-- internal stylesheet -->
+      <link rel="stylesheet" href="styles.css" />
+    </head>
+    <body>
+      <!-- jumbotron -->
+      <div class="bg-light p-5 rounded-lg">
+        <h1 class="display-4 text-center">MEET THE TEAM:</h1>
+        <h2 class="display-5 text-center">${teamNameAnswer.teamName.toUpperCase()}</h2>
+      </div>
+
+      <section>
+        <h2 class="text-center mt-1">Manager</h2>
+        <div class="card-container d-flex">
+        ${employees.managers.map(constructCard).join("")}
+        </div>
+      </section>
+
+      <section>
+        <h2 class="text-center">Engineers</h2>
+        <div class="card-container d-flex">
+        ${employees.engineers.map(constructCard).join("")}
+        </div>
+      </section>
+
+      <section>
+        <h2 class="text-center">Interns</h2>
+        <div class="card-container d-flex">
+        ${employees.interns.map(constructCard).join("")}
+        </div>
+      </section>
+    
+      <!-- Footer -->
+      <footer class="page-footer font-small blue">
+        <!-- Copyright -->
+        <div class="footer-copyright text-center py-3">
+          © 2021 Copyright:
+          <a href="https://github.com/lenny-g"> Lenny G</a>
+        </div>
+        <!-- Copyright -->
+      </footer>
+      <!-- Footer -->
+  
+      <!-- scripts JQuery -->
+      <script
+        src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
+        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
+        crossorigin="anonymous"
+      ></script>
+      <!-- scripts internal script -->
+      <script src="../src/lib/index.js"></script>
+    </body>
+  </html>
+  `;
+};
+
+const init = async () => {
+  let inProgress = true;
+  const managers = [];
+  const engineers = [];
+  const interns = [];
+
+  const teamNameAnswer = await inquirer.prompt(mainQuestion);
+
+  while (inProgress) {
+    const answers = await inquirer.prompt(questions);
+
+    if (answers.role === "manager") {
+      const manager = new Manager({
+        name: answers.name,
+        id: answers.id,
+        email: answers.email,
+        officeNumber: answers.officeNumber,
+      });
+      managers.push(manager);
+    }
+
+    if (answers.role === "engineer") {
+      const engineer = new Engineer({
+        name: answers.name,
+        id: answers.id,
+        email: answers.email,
+        github: answers.github,
+      });
+      engineers.push(engineer);
+    }
+
+    if (answers.role === "intern") {
+      const intern = new Intern({
+        name: answers.name,
+        id: answers.id,
+        email: answers.email,
+        school: answers.school,
+      });
+      interns.push(intern);
+    }
+
+    if (!answers.addAnother) {
+      inProgress = false;
+    }
+  }
+
+  const employees = { managers, engineers, interns };
+  console.log(employees);
+  const html = generateHTML(employees, teamNameAnswer);
+  writeToFile("./dist/index.html", html);
```

Comment:
I think the dist folder has been renamed which is why it's not generating your HTML. Consider changing this or adding a dist folder back in.